wps-cloudformation-template.yaml

```diff
@@ -57,7 +57,44 @@ Parameters:
   geoserverWpsEndpoint:
     Type: String
     Default: 'http://geoserver-wps.aodn.org.au/geoserver/wps'
+  environment:
+    Type: String
+    Default: dev
+    AllowedValues:
+      - production
+      - systest
+      - sandbox
+      - dev
+  createProdSystestBucket:
```

Comment:
This is an example of something that probably doesn't need to exist, see above. You've already told the template once which environment it is, so this flag is redundant.

Reply:
This flag will only be needed in two scenarios.

1. When we bring up production or systest environment for the first time.
2. If we accidentally delete systest/prod bucket and want to recreate it.

I just wanted to avoid the manual step of creating the bucket for any environment.